pkg/sfu/downtrack.go

```diff
@@ -645,8 +644,12 @@ func (d *DownTrack) FinalizeAllocate() VideoAllocation {
 	return d.forwarder.FinalizeAllocate(d.receiver.GetBitrateTemporalCumulative())
 }
 
-func (d *DownTrack) AllocateNextHigher() (VideoAllocation, bool) {
-	return d.forwarder.AllocateNextHigher(d.receiver.GetBitrateTemporalCumulative())
+func (d *DownTrack) AllocateNextHigher(availableChannelCapacity int64) (VideoAllocation, bool) {
+	return d.forwarder.AllocateNextHigher(availableChannelCapacity, d.receiver.GetBitrateTemporalCumulative())
+}
+
+func (d *DownTrack) GetNextHigherTransition() (VideoTransition, bool) {
```

Comment:
New API in down track to get how much the next jump needs and use that for probing with padding. With probing with media, we were using the method above this to switch to that layer and wait for congestion signal or not.